geomesa-core/src/main/scala/org/locationtech/geomesa/core/index/Strategy.scala

```diff
@@ -118,17 +117,16 @@ trait Strategy {
                      schema: String,
                      featureEncoding: FeatureEncoding,
                      query: Query): Option[IteratorSetting] = {
-    if (iteratorConfig.useSFFI) {
-      Some(configureSimpleFeatureFilteringIterator(featureType, ecql, schema, featureEncoding, query))
-    } else None
+    if (iteratorConfig.useSFFI) Some(configureSimpleFeatureFilteringIterator(featureType, ecql, schema, featureEncoding, query))
```

Comment:
ok, just so i know for future reference, are you referring to a style guide or is it just a convention in practice?